assets/training/finetune_acft_image/components/pipeline_components/hf_classification/spec.yaml

```diff
@@ -363,7 +363,7 @@ inputs:
 
   input_column_names:
     type: string
-    default: image_url
+    default: image
```

Comment:
Why are we changing the default image column name?

Reply:
To reduce confusion. This change should have been included in https://github.com/Azure/azureml-assets/pull/3016/ , but we missed it. LMK if it makes more sense for this to still be image_url - looks like it according to the existing code, https://github.com/Azure/azureml-assets/blob/main/assets/training/model_evaluation/src/image_dataset.py#L204 .

Reply:
Default in our image FT code is image_url --> 
We might need to update the notebook as well, if we change the col name.

Reply:
It seems the column name does not actually matter for the FT component. Also, whatever dataset we get from the outside (eg a notebook) is converted, so no need to change the column name. Will chat offline.

Reply:
NVM, this is default values. Sorry, I confused it.